src/test/java/de/tum/in/www1/artemis/PlagiarismCheckForExcercisesIntegrationTest.java

```diff
@@ -0,0 +1,83 @@
+package de.tum.in.www1.artemis;
+
+import de.tum.in.www1.artemis.domain.Course;
+import de.tum.in.www1.artemis.domain.enumeration.ExerciseType;
+import de.tum.in.www1.artemis.domain.plagiarism.PlagiarismComparison;
+import de.tum.in.www1.artemis.domain.plagiarism.PlagiarismResult;
+import de.tum.in.www1.artemis.domain.plagiarism.modeling.ModelingSubmissionElement;
+import de.tum.in.www1.artemis.domain.plagiarism.text.TextSubmissionElement;
+import de.tum.in.www1.artemis.util.FileUtils;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpStatus;
+import org.springframework.security.test.context.support.WithMockUser;
+import org.springframework.util.LinkedMultiValueMap;
+
+import java.io.IOException;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class PlagiarismCheckForExcercisesIntegrationTest extends AbstractSpringIntegrationBambooBitbucketJiraTest {
+
+    private Course course;
+
+    private int studentAmount;
```

Comment:
Codacy warns that this attribute is only used in one method and could be a local variable. Is this an attribute on purpose? Why?

Reply:
I removed the studentAmount

Reply:
The course is used in multiple methods, so we need to declare it as a global variable